source/OutfitterPanel.cpp

```diff
@@ -198,8 +198,8 @@ void OutfitterPanel::DrawItem(const string &name, const Point &point)
 	}
 	// Don't show the "in stock" amount if the outfit has an unlimited stock.
 	int stock = 0;
-	if(!outfitter.Has(outfit))
-		stock = max(0, player.Stock(outfit));
+	if(!outfitter.Has(outfit) && outfit->Get("installable") >= 0.)
```

Comment:
merge conflict artifact
```suggestion
	if(!outfitter.Has(outfit))
```